RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -218,6 +218,98 @@
                                     skipClusters = cms.InputTag('convClusters'),
                                     )
                                 )
+
+#this was done by Sam Harper (Aug 2017) and is a best guess (and I do guess wrong sometimes)
+#in general I kept all the old layer pairs and added sensible phaseI combinations
+#most pairs are adjacent layers (with some small exceptions) so I stuck to that
+from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
+trackingPhase1.toModify(convLayerPairs, layerList = cms.vstring(
+        'BPix1+BPix2', 
+        
+        'BPix2+BPix3', 
+        'BPix3+BPix4',#added for PhaseI
+        
+        'BPix2+FPix1_pos', 
+        'BPix2+FPix1_neg', 
+        'BPix2+FPix2_pos', 
+        'BPix2+FPix2_neg', 
+        'BPix2+FPix3_pos', #added for PhaseI (iffy)
+        'BPix2+FPix3_neg', #added for PhaseI (iffy)
+        
+        'BPix3+FPix1_pos', #added for phaseI (iffy)
+        'BPix3+FPix1_neg', #added for phaseI (iffy)
+        'BPix3+FPix2_pos', #added for phaseI (iffy)
+        'BPix3+FPix2_neg', #added for phaseI (iffy)
+        
+        'FPix1_pos+FPix2_pos', 
+        'FPix1_neg+FPix2_neg',
+        
+        'FPix2_pos+FPix3_pos', #added for phaseI 
+        'FPix2_neg+FPix3_neg',#added for phaseI 
+                
+        'BPix3+TIB1', 
+        'BPix3+TIB2', #maybe remove for phaseI?
+        'BPix4+TIB1', #added for phase I
+        'BPix4+TIB2', #added for phase I
+        
+        'TIB1+TID1_pos', 
```

Comment:
to clarify for this, you object to having two separate complete layer lists and would prefer the phase I customization to just add new layers?  

So I thought about that but figured in the end new detector, new layers list, I would prefer to keep them disabled, changes to the legacy pixels layers I dont think should be automatically propagated to the phase I pixels. 

Still in general you have a much better eye for this sort of thing and have more experience here in knowing what can go wrong so if you still want it done after justification I just gave then I'll do it.